editor/resources/templates/template.cp

```diff
@@ -0,0 +1,13 @@
+#version 430
+
+layout (local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
+
+layout(rgba32f) uniform image2D texture_out;
```

Comment:
Fix Inconsistent space?

Reply:
You resolved the conversation, but you didn't fix it? 😅 
```layout (```
vs
```layout(```

Reply:
I think I misunderstood what you were referring to, the file also contained a mix of space and tab indentations which I thought was the issue 😅 but thanks, I'll fix it!